pages/docs/community/tooling.md

```diff
@@ -42,6 +42,7 @@ The following is a list of tools that do not yet belong to any specific category
 | :------------- | :------------- | :----------------- |
 | [Converter](https://github.com/asyncapi/converter) | Converts old versions of AsyncAPI files into the latest version. | Javascript
 | [Converter Go](https://github.com/asyncapi/converter-go) | Converts old versions of AsyncAPI files into the latest version. Thanks to [@Kyma team](https://kyma-project.io/). | Go
+| [AsyncAPI-format](https://www.npmjs.com/package/asyncapi-format) | Format an AsyncAPI document by ordering, formatting, and filtering fields. | NodeJS
```

Comment:
@thim81 Hello! I just noticed that PR, sorry. Maybe you should give a link to your Github repository, not to npm, like other tools?

Reply:
@magicmatatjahu Hi, I updated the link to Github.